spec/presenters/queries/expanded_link_set_spec.rb

```diff
@@ -101,13 +101,13 @@
           a_hash_including(
             base_path: "/b",
             links: {
-            mainstream_browse_pages: [a_hash_including(
-              base_path: "/c",
-              links: {
-                parent: [
-                  a_hash_including(base_path: "/e", links: {})
-                ]
-              })]
+              mainstream_browse_pages: [a_hash_including(
+                base_path: "/c",
+                links: {
+                  parent: [
+                    a_hash_including(base_path: "/e", links: {})
+                  ]
+                })]
```

Comment:
Same point about stylistic changes in their own commits, although for these changes, the diff is easy to read, so it doesn't harm readability.